app/models/state_file_w2.rb

```diff
@@ -113,4 +115,22 @@ def state_tax_group_xml_node
     result = xml_template.at(:W2StateTaxGrp)
     result.nil? ? "" : result.to_xml
   end
+
+  def get_box14_ui_overwrite
+    box14_ui_wf_swf || box14_ui_hc_wd
+  end
+
+  private
+
+  def validate_box14_limits
+    validate_limit(:box14_ui_wf_swf, 179.78)
+    validate_limit(:box14_fli, 145.26)
+  end
+
+  def validate_limit(field, limit)
+    value = send(field)
+    if value.present? && value > limit
+      errors.add(field, I18n.t("validators.dollar_limit", limit: '%.2f' % limit))
+    end
+  end
```

Comment:
review comment: check if it would be cleaner to put this in the NJ-specific W2

Reply:
@mmazanec22 it looks like we don't actually have an NJ-specific W2 model yet. we could add one, but i think i will make a follow-up ticket for this